### DISCUSSION TITLE
終了テキスト

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1848] [Q&A] [answered]
出力の最後のテキストを削除することは可能ですか?

下の画像をご覧ください。

ありがとうございました。

![image](https://user-images.githubusercontent.com/49851974/182272223-b570761e-0c09-4502-9561-1d0e70bd10c1.png)

**Comment by kmuto** [comment, marked as answer]
config.ymlで
```
  …
pdfmaker:
  …
# colophon: null
  …
```

というところがあるはずなので、

```
  …
pdfmaker:
  …
  colophon: null
  …
```
とコメントを外してみてください。

**Reply by jasondee1992** [reply]
Thank you kmuto-san!
